.github/workflows/ci-python.yml

```
@@ -0,0 +1,67 @@
+name: Build and test Python package
+
+on:
+  push:
+    branches: [ master, ospd-21.04, ospd-20.08 ]
+  pull_request:
+    branches: [ master, ospd-21.04, ospd-20.08 ]
+
+jobs:
+  linting:
+    name: Linting
+    runs-on: 'ubuntu-latest'
+    strategy:
+        matrix:
+          python-version:
+            - 3.7
+            - 3.8
+            - 3.9
```

```suggestion
            - 3.5
            - 3.6
            - 3.7
```

ospd 20.8 is from 2020, so we provide support of python 3.5 and 3.6